packages/datatrak-web-server/src/utils/addRecentEntity.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import { DatatrakWebServerModelRegistry } from '../types';
+
+const MAX_RECENT_ENTITIES = 3;
+
+export async function addRecentEntity(models: DatatrakWebServerModelRegistry, userId: string, entityId: string) {
```

Comment:
Seems like it would be good to have a unit test for the util and that could be done without having to add too much boiler plate if we just mock out the models, right